### DISCUSSION TITLE
default value of rsvp_start_number_days and rsvp_start_number_hours

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#495] [General]
In the eme_events table, the event_properties field contains the rsvp_start_number_days and rsvp_start_number_hours properties. But I haven't found a way to set their default values, whereas I can do it for rsvp_end_number_days and rsvp_end_number_hours. Would it be possible to set these values other than using a filter?

**Comment by liedekef** [comment]
While this should be a feature request, see dev :-)

**Comment by f14m07** [comment]
in fact we have control when we set a new event, but we didn't have control over the default parameter. Thank you once again for this new development.
